jsapp/js/components/submissions/table.tsx

```diff
@@ -835,8 +835,12 @@ export class DataTable extends React.Component<DataTableProps, DataTableState> {
             if (Object.keys(TABLE_MEDIA_TYPES).includes(q.type)) {
               let mediaAttachment = null
 
-              if (q.type !== QUESTION_TYPES.text.id && q.$xpath !== undefined) {
-                mediaAttachment = getMediaAttachment(row.original, row.value, q.$xpath)
+              if (q.type !== QUESTION_TYPES.text.id && row.original._attachments[0]) {
+                mediaAttachment = getMediaAttachment(
+                  row.original,
+                  row.value,
+                  row.original._attachments[0].question_xpath,
```

Comment:
This change introduces a regression: audio and video files cannot be previewed: 

https://github.com/user-attachments/assets/d1ab6916-0eb8-4bcd-83fc-a4c865e92d66

It gets weirder if you create groups, redeploy, submit a new response, then delete those groups:

https://github.com/user-attachments/assets/6d89bf8f-197e-461c-9d51-4d7adf9f6e6b

^ the audio and video are playing the same file somehow